Cloud Application Manager/Deploying Anywhere/Using-CenturyLink-Private-Cloud-with-VMC-on-AW

```diff
@@ -0,0 +1,134 @@
+{{{ "title": "Using the CenturyLink Private Cloud with VMC on AWS",
+"date": "12-05-2019",
+"author": "Sharon Wang",
+"keywords": ["cam", "cloud application manager", "vmware", "AWS", "VMC"],
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+**In this article:**
+- [Overview](#overview)
+- [Audience](#audience)
+- [Prerequisites](#prerequisites)
+- [Required Permissions](#required-permissions)
+- [Bootstrapping virtual machine templates with the Cloud Application Manager Agent](#bootstrapping-virtual-machine-templates-with-the-cloud-application-manager-agent)
+- [Registering a VMC provider in Cloud Application Manager](#registering-a-vmc-provider-in-cloud-application-manager)
+- [Registering Instances to Your VMC Provider](#registering-instances-to-your-vmc-provider)
+- [Getting General Support](#getting-general-support)
+
+### Overview
+
+This article is meant to assist users of Cloud Application Manager to configure a CenturyLink Private Cloud with VMC on AWS provider, then deploy or register a virtual machines with it.
+
+### Audience
+
+All users of Cloud Application Manager using CenturyLink Private Cloud with VMC on AWS providers.
+
+### Prerequisites
+
+* An active *Cloud Application Manager* account
+* Access and credentials to a **CenturyLink Private Cloud with VMC on AWS**.
+* A virtual machine template (Linux or Windows) on the VMC. Virtual machine templates has to be bootstrapped with the Cloud Application Manager Agent.
```

Comment:
Hey @jinzishuai, is this accurate? Do we need the templates to be pre-populated?
```suggestion
* A virtual machine template (Linux or Windows) on the VMC. Virtual machine templates have to be bootstrapped with the Cloud Application Manager Agent.
```